README.md

```diff
@@ -74,8 +74,9 @@ https://cn.dapr.io/meeting/
 
 Every week, we meet to plan and sync on the progress for the milestone's release. This meeting includes maintainers and active contributors and usually lasts for about 30-60 minutes. If you would like to **actively contribute** towards a milestone, you are welcome to join that meeting too.
 
-**Zoom Call:** https://zoom.us/j/92337656846<br>
-**Passcode:** 815623<br>
+**Zoom Call:** https://zoom.us/j/91940016938<br>
```

Comment:
The community call meeting requires someone to login as host to start the call. It is too much formality for weekly sync when we not always have someone with access to the host creds.

Reply:
Agree on not requiring a host for weekly sync calls. Too much ceremony